internal/store/postgres/resource_repository_test.go

```diff
@@ -225,6 +225,14 @@ func (s *ResourceRepositoryTestSuite) TestGetOne() {
 	})
 }
 
+func (s *ResourceRepositoryTestSuite) TestGetResourcesTotalCount() {
+	s.Run("should return 0", func() {
+		_, actualError := s.repository.GetResourcesTotalCount(context.Background(), domain.ListResourcesFilter{})
+
+		s.Nil(actualError)
+	})
+}
+
```

Comment:
Can you also add a positive test case for `size` and `offset` changes?

Reply:
done